deployment/default.yml.example

```diff
@@ -9,7 +9,7 @@ NATInstanceAMI: ''
 BastionHostInstanceType: 't2.micro'
 BastionHostAMI: ''
 PrivateHostedZoneName: 'mmw.internal'
-PublicHostedZoneName: 'staging.app.wikiwatershed.org'
+PublicHostedZoneName: 'staging.app.modelmywatershed.org'
```

Comment:
Should this be `staging.modelmywatershed.org` instead of `staging.app.modelymywatershed.org`?

Reply:
Yes. Fixed in https://github.com/WikiWatershed/model-my-watershed/pull/3053/commits/643f4de5995ac85594d34e411aca0cfcff8e1fdf.